cmd/kogito/command/util/build_from_file.go

```diff
@@ -21,69 +21,123 @@ import (
 	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
 	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/message"
 	"io"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
-var supportedExtensions = []string{".dmn", ".drl", ".bpmn", ".bpmn2", ".properties"}
+var supportedS2iExtensions = []string{".dmn", ".drl", ".bpmn", ".bpmn2", ".properties"}
+var supportedJvmExtensions = []string{".jar", ".json"}
+var supportedNativeExtensions = []string{"-runner", ".json"}
+
+func zipFile(absoluteFilePath string, fileInfo os.FileInfo, resource string, binaryBuildType BinaryBuildType, tarWriter *tar.Writer) (string, error) {
```

Comment:
I believe that those IO/build related functions could be refactored to their own package instead of `util`.

Reply:
I moved this file and its test to an `iozip` package.